integration/deployment/deploy.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh -e
+DANMKUBECONFIG=/etc/kubernetes/danm.kubeconfig
+cp /opt/bin/* /target/cni-bin/
+cat >/target/netd/00-danm.conf <<-EOF
+{
+  "name": "meta_cni",
+  "type": "danm",
+  "kubeconfig": "$DANMKUBECONFIG"
+}
+EOF
+NAME=kubernetes
+HOST=$KUBERNETES_SERVICE_HOST
```

Comment:
HOST and PORT variables I don't see defined in the initcontainer part of netwatcher manifest
don't we need it?